src/org/labkey/test/tests/FolderExportTest.java

```diff
@@ -354,10 +354,12 @@ private void verifyProjectGroups(String projectName, @Nullable String[] groupsNo
     @LogMethod
     private void verifyCreateFolderFromTemplate()
     {
+        impersonateRole("Folder Administrator"); // Issue 52254
         _containerHelper.createSubFolderFromTemplate(getProjectName(), folderFromTemplate, "/" + getProjectName() + "/" + folderFromZip, new String[]{"Grid Views"});
```

Comment:
Be explicit about using the UI for this. `APIContainerHelper` (which `_containerHelper` is) currently uses the UI for this operation but it might be improved at some point and inadvertently lose this coverage.
```suggestion
        impersonateRole("Folder Administrator"); // Issue 52254
        new UIContainerHelper(this).createSubFolderFromTemplate(getProjectName(), folderFromTemplate, "/" + getProjectName() + "/" + folderFromZip, new String[]{"Grid Views"});
```